roles/aks-apply/files/dev/otp-data-builder-debug.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otp-data-builder-debug
+  labels:
+    app: otp-data-builder-debug
+    update: auto
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: otp-data-builder-debug
+  template:
+    metadata:
+      labels:
+        app: otp-data-builder-debug
+    spec:
+      containers:
+      - name: otp-data-builder-debug
+        image: ubuntu:24.04
+        # Sleep for a day because this is for debugging.
+        command: ["/bin/bash", "-c", "sleep 86400"]
```

Comment:
Maybe for this use case, using a job would be a better fit than a normal deployment. Then we could start it up when needed and it would automatically "close". Also, maybe staying on for a day is more than what is needed most times. Something like 30-60 mins should be enough.